orchestrator/cosmos_gravity/Cargo.toml

```diff
@@ -11,8 +11,9 @@ gravity_utils = {path = "../gravity_utils"}
 ethereum_gravity = {path = "../ethereum_gravity"}
 gravity_proto = {path = "../gravity_proto/"}
 
-deep_space = { git = "https://github.com/iqlusioninc/deep_space/", branch = "master" }
-ethers = { git = "https://github.com/iqlusioninc/ethers-rs.git", branch="zaki/error_abi_support", features = ["abigen"] }
+deep_space = { git = "https://github.com/crypto-org-chain/deep_space/", branch = "update/deps" }
```

Comment:
why was this upgraded?

Reply:
ok https://github.com/crypto-org-chain/deep_space/commit/979787732f5882969fde430f77b72fbd75ecf88f

Reply:
yeah, there were a lot of dependency mismatches